metpy/calc/thermo.py

```diff
@@ -535,33 +535,57 @@ def saturation_mixing_ratio(tot_press, temperature):
 
 
 @exporter.export
-@check_units('[pressure]', '[temperature]')
-def equivalent_potential_temperature(pressure, temperature):
+@check_units('[pressure]', '[temperature]', '[temperature]')
+def equivalent_potential_temperature(pressure, temperature, dewpoint):
     r"""Calculate equivalent potential temperature.
 
-    This calculation must be given an air parcel's pressure and temperature.
-    The implementation uses the formula outlined in [Hobbs1977]_ pg.78-79.
+    This calculation must be given an air parcel's pressure, temperature, and dewpoint.
+    The implementation uses the formula outlined in [Bolton1980]_:
+
+    First, the LCL temp is calculated:
+
+    .. math:: T_{L}=\frac{1}{\frac{1}{T_{D}-56}+\frac{(T_{K}/T_{D})}{800}}+56
+
+    Which is then used to calculate the potential temperature at the LCL:
+
+    .. math:: \theta_{DL}=T_{K}(\frac{1000}{p-e})^k(\frac{T_{K}}{T_{L}})^{.28r}
+
+    Both of these are used to calculate the final equivalent potential temperature:
+
+    .. math:: \theta_{E}=\theta_{DL}\exp[(\frac{3036.}{T_{L}}-1.78)\times r(1+.448r)]
```

Comment:
First term in the exponential needs the paren modification as well. Any thoughts on using `*` vs `\times`? The times symbol always makes me think cross-product, but that could be a special case of broken.